code/datums/uplink_item.dm

```diff
@@ -923,28 +906,27 @@ var/list/uplink_items = list()
 	name = "Syndicate Surgery Dufflebag"
 	desc = "The Syndicate surgery dufflebag is a toolkit containing all surgery tools, surgical drapes, a Syndicate brand MMI, a straitjacket, and a muzzle."
 	item = /obj/item/weapon/storage/backpack/dufflebag/syndie/surgery
-	cost = 3
+	cost = 2
 
 /datum/uplink_item/device_tools/military_belt
 	name = "Military Belt"
 	desc = "A robust seven-slot red belt that is capable of holding all items that can fit into it."
 	item = /obj/item/weapon/storage/belt/military
-	cost = 3
+	cost = 1
```

Comment:
3 is kinda steep for an obvious item like this and it suffers competition with toolbelts so yeah okay.